dnpm

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
I don't understand the purpose of this file and the `dnpx`.
It is clear what it does, but I'm not sure we should add it to the project.

Reply:
@Eomm considered so many issue i had while developing from scratch, i think a docker based envrioment is very neccessary. you might remember so many issues i had commented. for eg, even after i use nvm to install node10 on my system(ubuntu 1904), it will still crash suddently , i guess the kernel 5.x might caused this

Reply:
This is clear, but it is a problem due to a specific environment, pushing in the main repo is not useful for all the devs but only for a limited set.

Moreover, this file could be a potential security issue (if we don't add it to `npmignore` file), for this reason, I would prefer to remove it and, at least, add a script in the package json like [this one](https://github.com/fastify/fastify-redis/blob/4351265f29c43af9a564dc8e1dd3dbfbd5483c50/package.json#L8)